src/main/java/io/confluent/idesidecar/restapi/featureflags/HttpFlagEvaluationProvider.java

```diff
@@ -81,9 +84,11 @@ public void evaluateFlags(
           })
           .onFailure(failure -> {
             if (failure instanceof FeatureFlagFailureException) {
+              var level = testMode ? Logger.Level.DEBUG : Logger.Level.ERROR;
```

Comment:
Would it make sense to turn the log level into an application configuration that we can set in the [application.yml](https://github.com/confluentinc/ide-sidecar/blob/main/src/main/resources/application.yml) so that we can use different log levels for different modes/profiles? I feel we should not implement custom code for detecting the _current_ profile/mode.

Reply:
I found these log messages very distracting in tests, but they are important to have as errors in production. I'm not sure how to accomplish this by configuring the log levels for the whole application but not otherwise affecting something else.

Reply:
That makes sense. What about determining if we're running in test mode by checking if `io.quarkus.runtime.configuration.ConfigUtils.getProfiles()` contains `test` instead of verifying if `fetchUri` starts with `"http://localhost"`? I feel that's a bit more native to Quarkus.

Reply:
Thanks for the suggestion. I followed your suggestion and used `getProfiles()`.